tests/ert3/console/integration/test_cli.py

```diff
@@ -336,10 +264,42 @@ def test_cli_export_uniform_polynomial_evaluation(tmpdir):
     with unittest.mock.patch.object(sys, "argv", args):
         ert3.console.main()
 
-    _assert_uniform_export(workspace, "uniform_evaluation")
+    _assert_export(workspace, "uniform_evaluation")
 
 
-def test_cli_run_presampled(tmpdir):
+def _assert_distribution(workspace, experiment, distribution, coefficients):
+    samples = {idx: [] for idx in ("a", "b", "c")}
+    for sample in coefficients:
+        for key in ("a", "b", "c"):
+            samples[key].append(sample[key])
+
+    config = _load_experiment_config(workspace, experiment)
+    for p in config["parameters"]:
+        if p["distribution"]["type"] == distribution:
+            parameter = p
+            break
+
+    assert parameter
```

Comment:
Should we make this a bit more explicit? Perhaps we could declare `parameter` as `None` before the loop and then assert that it is not `None` after the loop?